src/drawers/elementDrawer.ts

```diff
@@ -60,6 +60,15 @@ export module _Drawer {
     public _getSelector(): string {
       return this._svgElement;
     }
+
+    public _getPixelPoint(selection: D3.Selection, datum: any, index: number): Point {
+      switch(this._svgElement) {
+        case "circle":
+          return { x: parseFloat(selection.attr("cx")), y: parseFloat(selection.attr("cy")) };
```

Comment:
The former would require some bit of architectural work beyond the scope of this PR.  I'll get to it, but beyond the scope of this pr.